hieradata/clients/agent.trusted.ci.jenkins.io.yaml

```diff
@@ -92,6 +92,8 @@ profile::updatecenter::rsync_privkey: >
   yhELTnLcIarOCyO2KKYf5WRC8XGvvlz70=]
 profile::buildagent::tools_versions:
   awscli: 2.28.3
+profile::aznfs::mounts:
+  "/updates-jenkins-io-data": "updatesjenkinsio.file.core.windows.net:/updatesjenkinsio/updates-jenkins-io-data"
```

Comment:
Syntax error: hotfix in https://github.com/jenkins-infra/jenkins-infra/commit/cac57d131c7387509d18ac23a29b42300957e779